<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-D2KIIH1" ignored="0" name="RegressionTest" tests="5" failures="5" timestamp="2022-02-19T13:50:32 CST" time="13.497" errors="0">
  <testcase name="CheckRememberMe_3" time="0.173" classname="com.salesforce.tests.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.salesforce.tests.LoginTest.CheckRememberMe_3(LoginTest.java:65)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- CheckRememberMe_3 -->
  <testcase name="ForgotPassword_4a" time="0.211" classname="com.salesforce.tests.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.salesforce.tests.LoginTest.ForgotPassword_4a(LoginTest.java:83)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- ForgotPassword_4a -->
  <testcase name="ForgotPassword_4b" time="0.03" classname="com.salesforce.tests.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.salesforce.tests.LoginTest.ForgotPassword_4b(LoginTest.java:98)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- ForgotPassword_4b -->
  <testcase name="LoginErrorMsg_1" time="0.029" classname="com.salesforce.tests.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.salesforce.tests.LoginTest.LoginErrorMsg_1(LoginTest.java:26)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginErrorMsg_1 -->
  <testcase name="LoginToSalesforce_2" time="0.028" classname="com.salesforce.tests.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.salesforce.tests.LoginTest.LoginToSalesforce_2(LoginTest.java:43)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginToSalesforce_2 -->
</testsuite> <!-- RegressionTest -->
